components/Firebird/Deployments/AccessCodeReminder.tsx

```diff
@@ -0,0 +1,73 @@
+"use client"
+
+import { getAccount } from "@/utils/firebird/api"
+import {
+  IFirebirdApiAccountResponse,
+  IFirebirdApiResponse,
+} from "@/utils/firebird/types"
+import { isLessThan24HoursAgo } from "@/utils/helpers"
+import { useEffect, useState } from "react"
+import { FaSpinner } from "react-icons/fa6"
+
+export const AccessCodeReminder = () => {
+  const [accountData, setAccountData] = useState<
+    IFirebirdApiResponse<IFirebirdApiAccountResponse>
+  >({
+    status: 0,
+    message: "",
+    data: {
+      created_at: 0,
+      email: "",
+      id: "",
+      origin: "",
+    },
+  })
+  const [loading, setLoading] = useState<boolean>(true)
+  const [error, setError] = useState<string | null>(null)
+  const isNewUser = isLessThan24HoursAgo(
+    accountData.data?.created_at,
+  ) as boolean
+
+  useEffect(() => {
+    const fetchAccount = async () => {
+      try {
+        const data = await getAccount()
+        setAccountData(data)
+      } catch (err) {
+        setError((err as Error).message || "An unknown error occurred")
+      } finally {
+        setLoading(false)
+      }
+    }
+
+    fetchAccount()
+  }, [])
+
+  if (loading) {
+    return (
+      <div>
+        <FaSpinner />
+      </div>
+    )
+  }
+
+  if (error) {
```

Comment:
This will look quite weird if it renders. There is a display error function you can use.